format-doc/scripts/format_markdown.py

```
@@ -0,0 +1,52 @@
+import sys
+import os
+import re
+
+def format_content(content):
+    # 見出しの後のバックスラッシュ付きピリオドなどを修正 (\. -> .)
+    content = re.sub(r'(\d+)\.', r'\1.', content)
+    # 見出しの前後などの余分な太字を整理（必要に応じて）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)